api-reference/v1.0/resources/groups_overview.md

```diff
@@ -0,0 +1,125 @@
+# Working with groups in Microsoft Graph
+
+Groups are collections of [users](user.md) and other principals who share access to resources in Microsoft services or in your app. Microsoft Graph provides you with APIs to create and manage a variety of different types of groups and group functionality to suit your scenario needs. All operations in Microsoft Graph on groups require administrator consent.
+
+> **Note**: Groups can only be created through work or school accounts, and are not supported for personal Microsoft accounts.
+
+| Type              | Use case | groupType | mail-enabled | security-enabled | Creation possible through API? |
+|-------------------|----------|-----------|--------------|------------------|--------------------------------|
+| Office 365 groups | Facilitating user collaboration with shared Microsoft online resources. | ["Unified"] | true | false | yes | [user](user.md) | 
+| Security groups | Controlling user access to in-app resources. | [] | false | true | yes |
+| Mail-enabled security groups | Controlling user access to in-app resources, with a shared group mailbox. | [] | true | true | no |
+| Distribution groups | Distributing mail to the members of the group. It is recommended to use Office 365 groups due to the richer set of resources it provides. | [] | true | false | yes |
```

Comment:
I don't think you can create DG/DLs through the API.  I think these need to be done through some Exchange UI or PowerShell.  So I think the answer here should be "No".